src/Components/Articles.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../utils/api";
+import Button from "react-bootstrap/Button";
+import { Link } from "react-router-dom";
+import Spinner from "react-bootstrap/Spinner";
+
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
This state setting probably isn't needed here, you could just set line 9 to initialise as true (instead of false), as this useeffect will only run once (due to having no dependencies on line 17). Hope that makes sense?